docs/user-guides/server_config_guide.md

```diff
@@ -0,0 +1,26 @@
+# Server Configuration Guide
+
+**Note:** For Installation and Starting of the Mumble Server, see the specific documentation for your Operating System ([Userguide Overview](README.md)).
+
+The Configuration involves three steps:
+1. Network configuration
+2. Main configuration file (`murmur.ini`)
+3. Administration via GUI
+
+
+## Network Configuration
+
+1. Static or dynamic IP:
+
+Your server is available over an IP.
+Depending on whether you 
+
+- Firewall & Ports
+
+## Configuration file
+
+[Murmur.ini Wiki](https://wiki.mumble.info/wiki/Murmur.ini)
+
+## Administration via GUI
+
+- Use the Mumble GUI for administration
```

Comment:
```suggestion
- Use the Mumble client GUI for administration
```

Reply:
Well, that is just placeholder for now, so there will hopefully at some point be a better description.